### DISCUSSION TITLE
Support of translating multiple branches in one Weblate component

### DISCUSSION THREAD

**Discussion by m-aciek** [WeblateOrg/weblate#5231] [Ideas]
If I understand well, current proposed approach is to set new components for feature branches. ([As per this FAQ answer.](https://docs.weblate.org/en/latest/faq.html#how-do-i-translate-several-branches-at-once))

Would you consider supporting such a user story?
* I push a new feature branch to my project's repository.
* Weblate pulls the changes on a feature branch to main repository (in case of changes in source strings, it keeps both strings, changed and from main branch).
* After translation weblate pushes commited translation to a feature branch.
* After merging the feature branch to main branch in project's repository, Weblate deletes old unused strings.

**Comment by nijel** [comment]
Weblate currently cannot merge content across branches. I think there was an issue to automatically create components for a feature branches and the translation propagation would then handle the consistency, but I can't find it right now....
